api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,486 @@
+swagger: '2.0'
+info:
+  title: Nakadi Event Bus API Definition
+  description: |
+    -------------------------------
+    Nakadi Event Bus API Definition
+    -------------------------------
+
+    * Low-level API
+    * Stantardised event format (see #/definitions/Event)
+    * Partition strategy: event store will infer the partition from the `partitioning_key` in the message body
+
+  version: '0.4'
+  contact:
+    name: Team Aruha @ Zalando
+    email: nakadi-users@zalando.de
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  oauth2:
+    type: oauth2
+    flow: implicit
+    authorizationUrl: 'https://auth.example.com/oauth2/tokeninfo'
```

Comment:
I think we should also authorize event bus access using scopes.
Would expect 2 scopes:  <stups-appl-id>.read and <stups-appl-id>.write